packages/debugger-extension/src/index.ts

```diff
@@ -48,6 +48,12 @@ import { Session } from '@jupyterlab/services';
 import { ISettingRegistry } from '@jupyterlab/settingregistry';
 import { ITranslator } from '@jupyterlab/translation';
 
+function notifyCommands(app: JupyterFrontEnd): void {
```

Comment:
Ah this seems to be triggering commands regardless whether the plugin is available or activated on the page.

Spotted downstream in Notebook 7, which does not include the command to inspect variables (`@jupyterlab/debugger-extension:variables`):

![image](https://user-images.githubusercontent.com/591645/219375471-e127e33d-544f-46c1-9556-ee11db35d712.png)

Reply:
Opened https://github.com/jupyterlab/jupyterlab/issues/14023